smac/epm/gaussian_process.py

```diff
@@ -57,21 +59,16 @@ def __init__(
         self,
         types: np.ndarray,
         bounds: typing.List[typing.Tuple[float, float]],
+        seed: int,
```

Comment:
Why do we switch back to forwarding integer seeds?

Reply:
Is the seed even used in here? If the GP deterministic given a seed?

Reply:
We don't switch back, we adapt the GP to follow the interface of the base_epm. And yes, the GP is deterministic give the seed.